### PR TITLE
New role: install_ca

### PR DESCRIPTION
**Pull request by cjeanner**
This PR has:
- [X] Appropriate testing (molecule, python unit tests)
- [X] Appropriate documentation (README in the role, main README is up-to-date)

### CODE REVIEW

File: ci_framework/roles/install_ca/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+- name: Install custom CA bundle from file
+  become: true
+  register: ca_bundle
+  when:
+    - cifmw_install_ca_bundle_src is defined
+    - cifmw_install_ca_bundle_src != ''
+  ansible.builtin.copy:
+    dest: "{{ cifmw_install_ca_trust_dir }}/{{ cifmw_install_ca_bundle_src | basename }}"
+    src: "{{ cifmw_install_ca_bundle_src }"
```
Comment (raukadah) [inline ci_framework/roles/install_ca/tasks/main.yml:42]: } is missing

File: ci_framework/roles/install_ca/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# All variables intended for modification should be placed in this file.
+# All variables within this role should have a prefix of "cifmw_install_ca"
+cifmw_install_ca_trust_dir: " /etc/pki/ca-trust/source/anchors/"
```
Comment (pablintino) [inline ci_framework/roles/install_ca/defaults/main.yml:20]: Be aware of the space here. " /..."

File: ci_framework/roles/install_ca/molecule/default/converge.yml
Comment (pablintino) [inline ci_framework/roles/install_ca/molecule/default/converge.yml:1]: Maybe adding an assert at the end that covers that the role effectively called update-ca-trust is useful, as now we can remove that call and the test still passes. Just asserting that /etc/pki/tls/certs/ca-bundle.crt changed.

**Pull request merged**